packages/routing-utils/src/schemas.ts

```diff
@@ -1,3 +1,101 @@
+export const mitigateSchema = {
+  description: 'Mitigation action to take on a route',
+  type: 'object',
+  additionalProperties: false,
+  required: ['action'],
+  properties: {
+    action: {
+      description: 'The mitigation action to take',
+      type: 'string',
+      enum: ['log', 'challenge', 'deny'],
+    },
+  },
+} as const;
+
+export const hasConditionValueSchema = {
+  description:
+    'A value to match against. Can be a string (regex) or a condition operation object',
+  anyOf: [
+    {
+      description: 'A string value (treated as regex)',
```

Comment:
```suggestion
      description: 'A regular expression used to match the value. Named groups can be used in the destination.,
```

I think the previous description was better. Lets revert this

Reply:
fixed.